types/express-boom/index.d.ts

```diff
@@ -0,0 +1,353 @@
+// Type definitions for express-boom 3.0
+// Project: https://github.com/scottcorgan/express-boom
+// Definitions by: Aurelien Gaston <https://github.com/Nox-404>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+/**
+ * Type definition for the middleware itself
+ */
+
+import express = require('express');
+declare namespace e { }
```

Comment:
What's the effect of merging in this empty namespace?

Reply:
As far as I know, this does nothing, so I don't strongly object to leaving it.

Reply:
I removed it, it seems to still does the trick so better getting rid of it !

Reply:
I didn't see this before I merged.  If you tag me on a follow-up PR, I'll fast-track it.